p4/include/tps_checksum.p4

```diff
@@ -19,29 +19,69 @@
 *************************************************************************/
 
 control TpsVerifyChecksum(inout headers hdr, inout metadata meta) {
-    apply {  }
+    apply {
+        verify_checksum(hdr.ipv4.isValid(),
+            {
+	        hdr.ipv4.version,
+	        hdr.ipv4.ihl,
+                hdr.ipv4.diffserv,
+                hdr.ipv4.totalLen,
+                hdr.ipv4.identification,
+                hdr.ipv4.flags,
+                hdr.ipv4.fragOffset,
+                hdr.ipv4.ttl,
+                hdr.ipv4.protocol,
+                hdr.ipv4.srcAddr,
+                hdr.ipv4.dstAddr
```

Comment:
Spacing is off.  maybe tabs and not spaces?

Reply:
Found and fixed with the telemetry report stuff. I'll fix this later.